app/views/scihist_image_viewer/_scihist_viewer_search_area.html.erb

```diff
@@ -27,7 +26,20 @@
 
 
       <div class="input-group-append">
-        <button type="submit" class="btn btn-brand-dark" title="Search" id="search-submit-header">
+        <%#
+          data-analytics-label is set to @work&.friendlier_id instead of @work.friendlier_id .
+          This is because spec/components/work_image_show_component_spec.rb renders this partial,
+          via app/components/work_image_show_component.html.erb, with a nil value for @work.
```

Comment:
Should we just fix the spec to pass in a work instead of allowing the mistake in real code to not have a work, if a work is always required to get good analytics among other things?